EventMixing/src/MixBackgroundFrames_module.cc

```diff
@@ -96,8 +92,7 @@ namespace mu2e {
 
     ProtonBunchIntensity pbi_;
     int totalBkgCount_;
-    int minSkip_;
-    int maxSkip_;
+    int skipFactor_;
```

Comment:
Why is this an integer and not a real number?  Perhaps I do not quite get the "philosophical"  meaning of this factor.   I understand the technical meaning and see that it is multiplied with real numbers in its use.

Reply:
I was just thinking of it as the number of mean microbunches you want to be able to skip in the background frame. I'm happy to change it to a real number or find a better name :)